project-description.md

```diff
@@ -0,0 +1,21 @@
+
+### Problem
+Images generated with text-to-image generators look more and more realistic. Large language models write increasingly complex texts and answer questions, so that they can hardly be distinguished from a human being. This not only creates tremendous potential to produce creative content and automating workflows, but also the danger of fake news, which can be generated with minimal effort. In order to counter this danger, it is necessary to develop new models that can reliably and securely detect AI-generated content and thus prevent fake news created by generative models.
+
+To approach the issue, we first conduct a technical market analysis. We examine existing AI models on the market that can be used for text and image generation. In addition, we want to evaluate which models can be used for the detection of AI-generated content and assess their performance. In the second step, we develope a completely new tool for AI content that reliably tricks existing AI detectors with little effort by making minimal changes to AI-generated texts or images. This can then be used during training of a new model for augmenting data to robustly train a new model against such modifications. For this reason, the software we develop shall be fast and reliable on images of arbitrary size and text of arbitrary length.
+
+### Task
+The task consists of two parts. In the first part we conduct a technical market analysis, where we investigate existing models that can be used for text and image generation, as well as which models and algorithms can be used for the detection of generated content. In the second part, we will develop a program that, by modifying AI-generated texts and images as minimally as possible, can outsmart existing AI detectors by making minimal changes to AI-generated texts and images.
+
+#### Technical market analysis
+The technical analysis will research generators for content generation by AI and programs for the detection of AI-generated content. The evaluation of the existing landscape for generative models should provide an overview of a domain that is developing very quickly and dynamically. The broadest possible range of software that can be used productively today should be considered, as well as an outlook on the latest research in this discipline. In addition to the analysis of generative models, we examine programs for the detection of AI-generated content and give an overview of existing products and current research. We evaluate the quality of the texts and images generated by the generative models and the quality of the detection of the regressive models. We use quantitative, qualitative or hybrid approaches to evaluate the performance of the software.
```

Comment:
What do you mean with "We use quantitative, qualitative or hybrid approaches to evaluate the performance of the software." ? i dont get it.

Reply:
> i think we should use always the same approach, so that we can compare the different software no?
I agree we should stick with the approach, when comparing different software.

> What do you mean with "We use quantitative, qualitative or hybrid approaches to evaluate the performance of the software." ?

It means that we maybe use quantitative approaches (like metrics that you can measure, e.g. the time a model needs to augment a picture) or qualitative approaches (like if the augmented picture is still close enough to the original picture) or a hybrid of both approaches to evaluate different existing software for the detection of AI-generated content.